decidim-admin/spec/commands/decidim/admin/publish_all_participatory_space_private_users_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Admin
+  describe PublishAllParticipatorySpacePrivateUsers do
+    subject { described_class.new(privatable_to) }
+
+    let!(:privatable_to) { create(:participatory_process) }
+    let!(:user) { create(:user, email: "my_email@example.org", organization: privatable_to.organization) }
+    let!(:private_user) { create(:participatory_space_private_user, user:, privatable_to:, role: { en: "Member" }, published: false) }
```

Comment:
confused me a bit .. maybe you could add some traits for that that `published` state?